docs/source/metrics.rst

```diff
@@ -1,6 +1,42 @@
 ignite.metrics
 ==============
 
+Metrics provide a way to compute various quantities of interest in an online
+fashion without having to store the entire output history of a model.
+
+In practice user needs to attach metric instance to engine. Metric value is
```

Comment:
nit: In practice **a** user needs to **the**  metric instance to **an** engine. **The** metric value is **then** computed using **the** output of the engine's `process_fn`: